cmd/fyne/internal/commands/flags.go

```diff
@@ -36,3 +38,191 @@ func (k *keyValueFlag) String() string {
 
 	return result
 }
+
+var stringFlags = map[string]func(*string) cli.Flag{
+	"app-id": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "app-id",
+			Aliases:     []string{"id"},
+			Usage:       "set app-id in reversed domain notation for android, darwin, and windows targets, or a valid provisioning profile for ios",
+			Destination: dst,
+		}
+	},
+	"app-version": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "app-version",
+			Usage:       "set version number in the form x, x.y or x.y.z semantic version",
+			Destination: dst,
+		}
+	},
+	"category": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "category",
+			Usage:       "macos: category of the app for store listing",
+			Destination: dst,
+		}
+	},
+	"certificate": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "certificate",
+			Aliases:     []string{"cert"},
+			Usage:       "ios/macos/windows: name of the certificate to sign the build",
+			Destination: dst,
+		}
+	},
+	"developer": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "developer",
+			Aliases:     []string{"dev"},
+			Usage:       "windows: the developer identity for your Microsoft store account",
+			Destination: dst,
+		}
+	},
+	"executable": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "executable",
+			Aliases:     []string{"exe"},
+			Usage:       "set path to executable",
+			DefaultText: "current directory main binary",
+			Destination: dst,
+		}
+	},
+	"icon": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "icon",
+			Usage:       "set name of the application icon file",
+			Destination: dst,
+		}
+	},
+	"install-dir": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "install-dir",
```

Comment:
Should this be "output" like used below?

Reply:
They mean different things though, so the description doesn't work in the same context.

Reply:
Absolutely. I just mean making the names consistent

Reply:
Given we have "src" this also could be "dst" instead, with a "d" alias.

Reply:
Currently in this branch shared flag names are global, which is why I'm hesitant to suggest "dir", as that would tie it to installation location. Otherwise would work well in this context.

Reply:
I like dst, sounds good